### ISSUE TITLE
Make README more clear and user-friendly

### ISSUE THREAD

**Issue by bonicim** [VOLTTRON/copier-poetry-volttron-agent#8]
The [current README](https://github.com/VOLTTRON/copier-poetry-volttron-agent) should be improved so that a user can quickly get started. Right now, there are issues installing the required dependency 'copier' which need to included in the README.   See https://github.com/copier-org/copier/issues/574

At the bare minimum the README should include:

* A high level description of the customer/user workflow; it should set expectations on what the user will get after using this tool
* A link to an Agent example repo of using this tool
* Prerequisites section with scripts and/or links to install any required software
* Documentation on configuring and using Github Workflows
* A Quick-start section for users who don't want to read the README and just want to start coding
* Recommended ways of using this tool (i.e. Docker environment, virtual machine, bare metal)

**Comment by bonicim** [comment]
A [bug relating to installing copier](https://github.com/copier-org/copier/issues/574) could potentially create a frustrating experience for adopters of this tool. Include a Prerequisite section that provides clear instructions on installing copier. Below is one example for installing copier on MacOS:

```md

# Prerequisites

MacOS:
* pipx install copier
* pipx inject copier "MarkupSafe<2.1.0"

```

**Comment by bonicim** [comment]
To confirm the scope of this issue, we are only updating the README for the copier repo. The README for the template (i.e. the readme under the project directory) is not part of the scope of this issue.

**Comment by kefeimo** [comment]
fixed by #41,
manually closed
